CenterForOpenScience/osf.io
# [OSF-8158] Remove harmful downcasing from BasicAuthProviderMixin

## Purpose

Downcasing urls for the BasicAuthProviderMixin prevented users working on a Fedora addon from using case sensitive urls, this fix removes unnecessary downcasing.

## Changes

Simple python fix removes .lower method.

## Side effects

None that I know of.

## QA Notes
Probably will want to test authentication with the various addons, especially if you can try out what happens when you use mixed case on all providers, whether they expect to have mixed case or not.

## Ticket

https://openscience.atlassian.net/browse/OSF-8158
